packages/fx-core/resource/package.nls.json

```diff
@@ -207,6 +207,8 @@
   "error.generator.DownloadSampleApiLimitError": "Unable to download sample due to throttling. Retry later after rate limit reset (This may take up to 1 hour). Alternatively, you can go to %s to git clone the repo manually",
   "error.generator.DownloadSampleNetworkError": "Unable to download sample due to network error. Check your network connection and retry. Alternatively, you can go to %s to git clone the repo manually",
   "error.generator.ParseUrlError": "Unable to parse url %s",
+  "error.copilotPlugin.openAiPluginManifest.CannotGetLogoError": "Unable to get logo from '%s'. Please update '%s' manually if you want to use same logo for your Copilot plugin.",
+  "error.copilotPlugin.openAiPluginManifest.CannotGetManifest": "Unable to get OpenAI plugin manifest from '%s'.",
   "error.m365.NotExtendedToM365Error": "Unable to extend Teams app to Microsoft 365. Use 'teamsApp/extendToM365' action to extend your Teams app to Microsoft 365.",
   "core.QuestionAppName.validation.pattern": "Application name must start with letters and contain at least two letters or digits. It can not contain some special characters.",
```

Comment:
When would we pop up this error message?

Reply:
@kuojianlu could you please help on this - when will "error.m365.NotExtendedToM365Error" show to users?

Reply:
When users try preview a M365 app in Outlook/Office but the app has not been extended to M365.